.env

```diff
@@ -1,11 +1,12 @@
-# Shared environment variables. Add env variables specific to your environment in .env.development.local
+# Shared environment variables. Add env variables specific to your environment in .env
 
 REACT_APP_API_URL=http://localhost:4000
 REACT_APP_JORE_API_URL=https://dev.kartat.hsl.fi/jore/graphql
 REACT_APP_CLIENT_ID=7833861618225795
-REACT_APP_CLIENT_SECRET=
 REACT_APP_REDIRECT_URI=http://localhost:3000
-HSL_TESTING_HSLID_USERNAME=
```

Comment:
These secrets should be removed after applying this change, so that no hanging secrets exist.

Reply:
Removed from all envs